homeassistant/components/immich/services.yaml

```diff
@@ -0,0 +1,15 @@
+upload_file:
+  fields:
+    config_entry_id:
+      required: true
+      selector:
+        config_entry:
+          integration: immich
+    file:
+      required: true
+      selector:
+        text:
```

Comment:
I believe we have a file selector, let me check what is best here

Reply:
there is no defined file [selector](https://www.home-assistant.io/docs/blueprint/selectors/)

Reply:
2f800ecc5e70a64b75580718ad344f19970e39e2